### PR TITLE
107 toggle not available property for drop off locations

### PR DESCRIPTION
**Pull request by bdey05**
## Description of this change
The following change adds a Checkbox to the “Drop-off Location Organizer” which allows the ability to change the “Not Available” property in the “📍 Drop off locations” table in Airtable. When this Checkbox is checked, it triggers an Airtable automation that automatically removes the checked Drop-off Location from all drivers for the next upcoming event in the “📅 Scheduled Slots” table. It also clears the "Starts accepting at" and "Stops accepting at" properties for the selected Drop-off Location in the “📍 Drop off locations” table.
### Before this PR:
Before
### After this PR:
After

### CODE REVIEW

File: client/src/pages/DriverLocationInfo/DropoffOrganizerPopup.tsx [hunk 1]
```diff
@@ -49,9 +104,56 @@ function processDropoffLocationsForTable(
...
+          >
+            <RadixCheckbox.Indicator className="CheckboxIndicator">
+              <img src={check_icon} alt="" />
+            </RadixCheckbox.Indicator>
+          </RadixCheckbox.Root>
+        )}
+      </div>
+    );
+  };
+
```
Comment (jasoncavanaugh) [inline client/src/pages/DriverLocationInfo/DropoffOrganizerPopup.tsx:151]: Unless I'm mistaken, this component is identical to the `HttpCheckbox` component that you are importing. If I were you, I would move that component into the `components` directory and import it in both places from there.

Comment (bdey05) [reply client/src/pages/DriverLocationInfo/DropoffOrganizerPopup.tsx]: You are right. I created the HttpCheckbox component in the components directory.

**Comment by mattsahn** [comment]
> This PR was not deployed automatically as @bdey05 does not have access to the Railway project.
> 
> In order to get automatic PR deploys, please add @bdey05 to the project inside the [project settings page](https://railway.app/project/031d77ce-b4ed-44da-afd3-6fad256781dc/settings/members).

@bdey05, it looks like you accepted the invite to Railway. I think you'll need to cancel and resubmit as a new PR in order to trigger the auto-deploy to happen.

**Comment by bdey05** [comment]
@mattsahn I reopened the PR and it looks like it triggered the Railway auto-deploy.

**Comment by mattsahn** [comment]
> @mattsahn I reopened the PR and it looks like it triggered the Railway auto-deploy.

Awesome, will take a look.

**Comment by jasoncavanaugh** [comment]
The code changes look fine for the most part, but I don't know if the feature is working as intended? When I mark a location as "Not Available", it doesn't remove the location from the Location Information table. Additionally, if I assign a driver to the location, and then remove the location, it doesn't update the driver to show that they are no longer assigned to that specific location. Is the Airtable automation supposed to take care of this?

**Comment by mattsahn** [comment]
> The code changes look fine for the most part, but I don't know if the feature is working as intended? When I mark a location as "Not Available", it doesn't remove the location from the Location Information table. Additionally, if I assign a driver to the location, and then remove the location, it doesn't update the driver to show that they are no longer assigned to that specific location. Is the Airtable automation supposed to take care of this?

Yeah, looks like the Airtable automation hasn't been added to the prod database yet. So, this is just the code change to toggle the "Not Available" field, which looks to be working fine. I'll approve this one and merge since that code is working and @bdey05 and @hgreenhut you'll need to followup on adding the automation to prod airtable separately.

**Review by mattsahn — APPROVED** [review]
looks good to me for code change to toggle drop off location field.

**Pull request merged**